scrapi/migrations.py

```diff
@@ -36,10 +39,58 @@ def rename(docs, target=None, **kwargs):
             tasks.process_normalized(tasks.normalize(raw, raw['source']), raw)
             logger.info('Processed document from {} with id {}'.format(doc.source, raw['docID']))
 
-            es.delete(index=settings.ELASTIC_INDEX, doc_type=doc.source, id=raw['docID'], ignore=[404])
-            es.delete(index='share_v1', doc_type=doc.source, id=raw['docID'], ignore=[404])
+            ElasticsearchProcessor.manager.es.delete(index=settings.ELASTIC_INDEX, doc_type=doc.source, id=raw['docID'], ignore=[404])
+            ElasticsearchProcessor.manager.es.delete(index='share_v1', doc_type=doc.source, id=raw['docID'], ignore=[404])
+
+        logger.info('Renamed document from {} to {} with id {}'.format(doc.source, target, raw['docID']))
+
+
+@tasks.task_autoretry(default_retry_delay=30, max_retries=5)
+def cross_db(docs, target_db=None, index=None, **kwargs):
+    """
+    Migration to go between cassandra > postgres, or cassandra > elasticsearch.
+    TODO - make this source_db agnostic. Should happen along with larger migration refactor
+    """
+    assert target_db, 'Please specify a target db for the migration -- either postgres or elasticsearch'
+    assert target_db in ['postgres', 'elasticsearch'], 'Invalid target database - please specify either postgres or elasticsearch'
+
+    for doc in docs:
+
+        if not doc.doc:
+            # corrupted database item has no doc element
+            logger.info('Could not migrate document from {} with id {}'.format(doc.source, doc.docID))
+            continue
+
+        raw = RawDocument({
+            'doc': doc.doc,
+            'docID': doc.docID,
+            'source': doc.source,
+            'filetype': doc.filetype,
+            'timestamps': doc.timestamps,
+            'versions': doc.versions
+        }, validate=False)
+
+        normed = util.doc_to_normed_dict(doc)
+
+        # Create the normalized, don't validate b/c its been done once already
+        normalized = NormalizedDocument(normed, validate=False)
+
+        if target_db == 'postgres':
```

Comment:
why not use the processing function ```get_processor(processor_name)```?
Plus that way you don't need to import them up top.